### ISSUE TITLE
Is there any way to modify the constraints?

### ISSUE THREAD

**Issue by zidanessf** [joaquimg/BilevelJuMP.jl#26]

**Comment by joaquimg** [comment]
Currently no.
But JuMP has methods for that! We can extend them.
Are you used to JuMP 0.21.x?

**Comment by zidanessf** [comment]
no, I'm using JuMP v0.20.1. The "delete" method from JuMP simply sets the state of the constraint to invalid. Since adding one constraint to the bilevel problem generates some variables and constraints to its single-level counterpart, we can build a tree to trace these var/cons and delete them when needed.
